src/s2_pn_counters.erl

```diff
@@ -34,7 +34,7 @@
 %%%_* Code =============================================================
 %%%_ * Types -----------------------------------------------------------
 -define(M                 ,  s2_vclocks).
--opaque pn_counter()      :: {?M:new(), ?M:new()}.
+-opaque pn_counter()      :: {?M:vclock(), ?M:vclock()}.
```

Comment:
Why this change in this PR?

Reply:
In order to make the CI work for all OTP versions, dialyzer would complain for OTP26 and OTP27 because of the unknown type. I'll change the description a bit to emphasise getting the library to work on later OTP versions.